docs/servers/ssh-into-server.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Connect to your servers using SSH
+intro: Sometimes you and your team members need to connect to your servers to run commands from the terminal. Connect via SSH to ensure your servers only executes commands by authorized SSH keys.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+        - /docs/ssh-keys/remove-ssh-key
+    guides:
+    related:
+        - /docs/ssh-keys/add-ssh-key
+        - /docs/ssh-keys/remove-ssh-key
+    featured:
+---
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Servers` card
+1. Click the `View` in the `Servers` card to see a list of existing `Servers`
+1. In the list of servers, find the server you want to connect and copy the server' IP Address
+    > The ip address should be number separated by dots, e.g: 192.0.2.1
```

Comment:
We should not give local IP addresses (`192.0.2.1`) as an example of remote server's IP address.
```suggestion
    > The ip address should be a number separated by dots, e.g: 41.234.56.78
```